closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -469,39 +493,54 @@ int decryptFinal(byte[] in, int inOfs, int len,
                      byte[] out, int outOfs)
         throws IllegalBlockSizeException, AEADBadTagException,
         ShortBufferException {
+        int bufSize = 0;
+        byte[] keyCopy;
+        byte[] ivCopy;
+        byte[] aad;
+        int tagLenBytes = this.tagLenBytes;
```

Comment:
This must be in the `synchronized` block; don't reuse field name.

Reply:
Done.